sources/packages/api/src/services/student-scholastic-standings/student-scholastic-standings.models.ts

```diff
@@ -0,0 +1,9 @@
+import { ScholasticStandingStatus } from "../../database/entities";
+
+/**
+ * Service model to fetch Pending And Denied scholastic standings.
+ */
+export interface PendingAndDeniedScholasticStandings {
```

Comment:
This model represents ScholasticStandings, we can detach the status from the interface name. If it is very basic model then we can prefix it by Basic or something like that IMO.

Reply:
That is the logic of service to extract only retrieve ScholasticStandings of a certain status. But ScholasticStanding is a model which respresents the transformed entity for service.

Reply:
As we talked @ann-aot, I can see why it was first done in this way but the reasons do not exist anymore because of the mapping on the controller hence it would be good t change as @dheepak-aot mentioned.

Reply:
function is updated, please have a look